sdk/react/src/views/AdPlaybackScreen/AdPlaybackScreen.js

```diff
@@ -56,7 +57,8 @@ export default class AdPlaybackScreen extends Component {
   };
 
   state = {
-    showControls: true
+    showControls: true,
```

Comment:
`showControls` is not modified anywhere and is always `true`, so it's redundant

Reply:
done